doc/initialize_root_password_howto.md

```diff
@@ -0,0 +1,32 @@
+# Initialize root user password HOWTO
+
+The password is empty for the `root` user when deploying a StarRocks cluster from fresh installation. This can be a security concern. This document describes steps to initialize root password from fresh installation.
+
+In the following examples, `mysql_password` is taken as the password for the root user, it can be replaced with any password chosen for the root.
+
+## Prerequisites
+
+Deploy StarRocks cluster using Helm and custom configuration.
+
+## Steps to Change the Password For root User
+
+1. Add the Helm Chart Repo for StarRocks. The Helm Chart contains the definitions of the StarRocks Operator and the custom resource StarRocksCluster.Refer to: https://docs.mirrorship.cn/en-us/latest/deployment/helm#procedure
```

Comment:
link to starrocks github repository. (https://github.com/StarRocks/starrocks/blob/main/docs/deployment/helm.md)

Reply:
done